web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html

```diff
@@ -1,7 +1,9 @@
 <!-- START NEW MAPPING VIEW -->
 <div *ngIf="isVersionCompatibleWithES" class="source-prop-container-map-exp">
-
-  <div>
+  <div *ngIf="targetEntity && !targetEntity.hasDocs">
+    <em>Unable to find documents that match your query.</em>
```

Comment:
@eltesoro-ml has updated the messaging, see the Jira ticket for the new version.

Reply:
Updated the message as per the Jira tifcket.